src/MailingListBundle/Block/Service/ContactsManager.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace Opifer\MailingListBundle\Block\Service;
+
+use Symfony\Component\DependencyInjection\ContainerInterface as Container;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\HttpKernel\Log\LoggerInterface;
+use Opifer\MailingListBundle\Entity\MailingList;
+use GuzzleHttp\Client as GuzzleClient;
+use GuzzleHttp\HandlerStack;
+use GuzzleHttp\Subscriber\Oauth\Oauth1;
+
+class ContactsManager
+{
+	/** @var string */
+    protected $consumer_key;
+
+    /** @var string */
+    protected $consumer_secret;
+
+    /** @var Service Container */
+    protected $service_container;
+
+    /** @var LoggerInterface */
+    protected $logger;
+
+    public function __construct(EntityManagerInterface $em, Container $container, LoggerInterface $logger, $consumer_key, $consumer_secret)
```

Comment:
There's no need to inject the complete service container. (Especially because you don't even use it)